graylog2-web-interface/src/components/notifications/GlobalAppNotifications.tsx

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Even though I used `git mv` before I applied the component changes, the reference got lost. If you think it is worth the effort I can give it another try.

Reply:
I do not think this will do much since we do a squash later on and git can keep so much..